cpp/include/cudf/io/parquet.hpp

```diff
@@ -410,6 +410,7 @@ class parquet_reader_options_builder {
    *
    * @param val Boolean value whether to read matching projected and filter columns from mismatched
    * Parquet sources.
+   *
```

Comment:
For the reviewer: I couldn't find this function in the docs, so I thought it may not be rendering correctly without this new line. I confirmed that you can see the function in the docs with it.

Reply:
Wow, thanks for catching this! We usually don’t include a blank line between `@param` and `@return`, but it might be time to change that.